From de5bb1203f904e136721e4f076836eaf9868e889 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Sun, 06 Mar 2022 17:31:06 +0200
Subject: [PATCH] [SW-69581] gaudi3: adjust strap reg name to latest specs

Strap registers were relocated in specs headers, and the names
were changed. Code is adjusted respectively.

Change-Id: I86867b74d16df7143885792057ff9fc8cc49f3d7
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 4883ebe..6aed290 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -4,6 +4,7 @@
  */
 
 #include <d0_psoc_global_conf_regs.h>
+#include <d0_psoc_boot_conf_regs.h>
 #include <d0_cpu_timestamp_regs.h>
 #include <d0_cpu_ca53_cfg_regs.h>
 #include <d0_psoc_gpio0_regs.h>
@@ -114,7 +115,7 @@
 
 bool hl_is_debug_mode(enum hl_debug_hw_func func)
 {
-	u32 boot_strap = regmap_readl(mmD0_PSOC_GLOBAL_CONF_BOOT_STRAP_PINS_L);
+	u32 boot_strap = regmap_readl(mmD0_PSOC_BOOT_CONF_BOOT_STRAP_PINS_L);
 	u32 strap_mask;
 
 	/* TODO: verify relevant straps for debug functions */
